packages/mwp-tracking-plugin/src/util/avro.js

```diff
@@ -101,22 +101,29 @@ const activity = {
 type Serializer = Object => string;
 type Deserializer = string => Object;
 
-const avroSerializer: Object => Serializer = schema => data => {
-	const record = avro.parse(schema).toBuffer(data);
-	// data.timestamp _must_ be ISOString if it exists
-	const timestamp = data.timestamp || new Date().toISOString();
-	const analytics = {
-		record: record.toString('base64'),
-		schema: `gs://meetup-logs/avro_schemas/${schema.name}_${schema.doc}.avsc`,
-		date: timestamp.substr(0, 10), // YYYY-MM-DD
+const avroSerializer: Object => Serializer = schema => {
+	const codec = avro.parse(schema);
```

Comment:
this is the main optimization - the codec created by `avro.parse(schema)` is now instantiated once when the module is first loaded rather than once-per-request, which turns out to be expensive